packages/snjs/lib/services/features_service.ts

```diff
@@ -86,9 +88,11 @@ export class SNFeaturesService extends PureService<void> {
     }
   }
 
-  public async loadUserRoles(): Promise<void> {
+  public async loadUserRolesAndFeatures(): Promise<void> {
     this.roles =
       (await this.storageService.getValue(StorageKey.UserRoles)) || [];
+    this.features =
+      (await this.storageService.getValue(StorageKey.UserFeatures)) || [];
```

Comment:
I believe getValue has a defaultValue param, if it makes things easier.

Reply:
Good catch, I've updated it 🙂